lib/cartopy/tests/mpl/test_gridliner.py

```diff
@@ -365,3 +367,40 @@ def test_gridliner_line_limits():
     for path in paths:
         assert (np.min(path.vertices, axis=0) >= (xlim[0], ylim[0])).all()
         assert (np.max(path.vertices, axis=0) <= (xlim[1], ylim[1])).all()
+
+
+@pytest.mark.natural_earth
+@ImageTesting([grid_label_intersect_map_image],
```

Comment:
Do we need another 1 MB image comparison test? Or can this be checked by comparing against the positions of the labels and whether they are visible or not?

Or maybe just a single smaller image would be fine.

Reply:
Yes that could work. I'll make some new test tomorrow.